### ISSUE TITLE
os.system in config module

### ISSUE THREAD

**Issue by alevoski** [alevoski/decontamine_Linux#2]
**To improve**
There are 3 system calls with os.system() function used to sed a file. It's dangerous : it uses the shell !

**Solution**
Replace them by subprocess.

**Alternative**
Find a pythonic way to sed a file

**Comment by will-tam** [comment]
Where ?

**Comment by alevoski** [comment]
Here, in config.py :
```
122:                os.system("sed -i '/" + tool + "/{n;n;s/.*/version = " + str(version).replace('/', '\/') + "/}' " + cfgFile)
242:                        os.system("sed -i '/" + tool + "/{n;n;s/.*/version = " + str(version).replace('/', '\/') + "/}' " + filepath)
245:                    os.system("sed -i '/" + tool + "/{n;s/.*/active = " + str(toMod) + "/}' " + filepath)
```

**Comment by alevoski** [comment]
Solved with commit c594325.
